base/commandLine.cpp

```diff
@@ -306,6 +306,7 @@ void registerDefaults() {
 #endif
 #if (defined(ENABLE_SCUMM) && defined(ENABLE_SCUMM_7_8)) || defined(ENABLE_GRIM)
 	ConfMan.registerDefault("dimuse_tempo", 10);
+	ConfMan.registerDefault("dimuse_v2_tempo", 50);
```

Comment:
change to 50 above. it was lowered in past for slow devices, can be restored now

Reply:
Very good. I will also put a piece of code to update the tempo to the new one if the current config file has the old one.